imap_processing/cdf/config/imap_glows_l1b_variable_attrs.yaml

```diff
@@ -0,0 +1,336 @@
+int_fillval: &int_fillval -9223372036854775808
+valid_max: &valid_max 3155630469184000000
+
+epoch_attrs:
```

Comment:
I commented on @bourque's PR as well because I thought we were doing a 1-1 mapping of variable names to the yaml attrs definitions, but now I see this might be the convention. I'm also confused though because there are also some variables below without the `_attrs` portion. Is there a convention we should be following?

Reply:
Here, the `_attrs` indicate dimensions, but that is ... not a good way of doing that, so I'll rename them. But there aren't really any conventions yet for this documents, so it's a little bit wild west...